src/Event/Event.php

```diff
@@ -64,7 +65,9 @@
 use CultuurNet\UDB3\Media\Image;
 use CultuurNet\UDB3\Model\ValueObject\Calendar\SubEventUpdate;
 use CultuurNet\UDB3\Model\ValueObject\MediaObject\CopyrightHolder;
+use CultuurNet\UDB3\Model\ValueObject\MediaObject\Video;
 use CultuurNet\UDB3\Model\ValueObject\Taxonomy\Label\Labels;
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID as ModelUUID;
```

Comment:
Nit: Preferable we'd alias the old `UUID` class instead, so when we remove that one later we don't need to clean up this alias on the new class later.

Reply:
Good point! I replaced all usages:
```
use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
use ValueObjects\Identity\UUID as LegacyUUID;
```